recipes/ppci/meta.yaml

```diff
@@ -0,0 +1,41 @@
+{% set name = "ppci" %}
+{% set version = "0.5.4" %}
+{% set   sha256 = "d5400cc043ca3eba0063f5dc4025747352fb03f950bd4b21e67bfa3f633966a3" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  noarch: python
```

Comment:
Does this package support python 2? The metadata in the `setup.py` only mentions python 3. Looking at `ppci / __init__.py` it looks like it's python 3 only. If this is the case, you need to drop the `noarch: python` and add `skip: True  # [py2k]`

Reply:
Yes, it is python 3 only.